chromium_src/chrome/common/chrome_features.cc

```diff
@@ -14,7 +14,7 @@
 namespace features {
 
 OVERRIDE_FEATURE_DEFAULT_STATES({{
-    {kHttpsFirstModeV2, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kHttpsFirstModeV2, base::FEATURE_ENABLED_BY_DEFAULT},
```

Comment:
We can enable most of them -- a number are still broken because we are modifying the HTTPS Upgrades feature. I pushed a commit that fixes some tests and narrowly allowlists the rest.